ecosystem/plugin-docsearch/src/client/composables/useDocsearchHotkeyListener.ts

```diff
@@ -5,10 +5,15 @@ import { useEventListener } from '@vueuse/core'
  */
 export const useDocsearchHotkeyListener = (callback: () => void): void => {
   const remove = useEventListener('keydown', (event) => {
-    if (event.key === 'k' && (event.ctrlKey || event.metaKey)) {
-      event.preventDefault()
-      callback()
-      remove()
+    const isHotKeyBind = event.key === 'k' && (event.ctrlKey || event.metaKey)
+    const isSlashKey = event.key === '/'
+
+    if (!isSlashKey && !isHotKeyBind) {
+      return
     }
+
+    event.preventDefault()
+    callback()
+    remove()
```

Comment:
BTW, I do not like the changed here, 
```suggestion
    if (isSlashKey || isHotKeyBind) {
      event.preventDefault()
      callback()
      remove()
    }
```

```suggestion
    const isHotKeyBind = (event.key === 'k' && (event.ctrlKey || event.metaKey))
    const isSlashKey = event.key === '/'

    if (isSlashKey || isHotKeyBind) {
      event.preventDefault()
      callback()
      remove()
    }
```

Reply:
I understand your point, but the "early return" is a very considerable and advantageous approach to make the logic clearer and more evident (indicated when talking about clean code). This function will not do anything else if the hotkeys are not triggered, so this way avoids the need to put the entire logic of the method inside the if.